src/langs/de.json

```diff
@@ -35,6 +35,7 @@
   "INVITATIONS_LOADING_MESSAGE": "Wir warten auf Ihre Einladung, bitte warten …",
   "USERNAME_TOO_SHORT_ERROR": "Bitte geben Sie einen Benutzernamen mit mehr als {{min}} Zeichen ein",
   "USERNAME_TOO_LONG_ERROR": "Bitte geben Sie einen Benutzernamen mit weniger als {{max}} Zeichen ein.",
+  "USERNAME_SPECIAL_CHARACTERS_ERROR": "Der Benutzername darf nicht enthalten \" \", \", <, >, ^, %, \\",
```

Comment:
Thanks for translating aha, we actually have a platform to do it (especially arabic). I appreciate if you do spanish (perhaps italian) since you speak it, but don't feel pressured to do german^^

Reply:
👍🏾